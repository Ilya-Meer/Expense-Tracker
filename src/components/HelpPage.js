import React, { Component } from 'react';

const HelpPage = () => (
  <div><p>I am the Help Page route!</p></div>
);

export default HelpPage;