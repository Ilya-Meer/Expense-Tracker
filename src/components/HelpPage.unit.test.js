import React from 'react';
import { shallow } from 'enzyme';
import HelpPage from './HelpPage';

describe('Help Page', () => {
  const wrapper = shallow(<HelpPage />);

  it('renders properly', () => {  
    expect(wrapper).toMatchSnapshot();
  })
})
