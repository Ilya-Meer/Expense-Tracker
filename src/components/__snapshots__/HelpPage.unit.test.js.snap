// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help Page renders properly 1`] = `
<div>
  <p>
    I am the Help Page route!
  </p>
</div>
`;
